dotcom-rendering/src/web/lib/verticalDivider.ts

```diff
@@ -12,6 +12,7 @@ export const verticalDivider = css`
 			left: 0;
 			width: 1px;
 			height: 100%;
+			height: calc(100% - var(--card-divider-offset-bottom));
```

Comment:
Is there a situation where `--card-divider-offset-bottom` is not defined and if so how does it behave?

I'm very cautious here that using a CSS variable like this introduces a 'hidden' relationship between components, is there any way we can avoid this? If not we should definitely add a comment explaining exactly what this is doing and where it's set!